workflow/commons/src/main/java/com/datastax/oss/dsbulk/workflow/commons/log/LogManager.java

```diff
@@ -241,9 +252,12 @@ public void reportAvailableFiles() throws IOException {
       LOGGER.info(
           "Errors are detailed in the following file(s): {}", Joiner.on(", ").join(debugFiles));
     }
-    if (!resources.isEmpty()) {
-      writeSummaryFile();
-      LOGGER.info("A summary of the operation in CSV format can be found in {}.", SUMMARY_CSV);
+    if (checkpointEnabled) {
+      CheckpointManager manager = mergeCheckpointManagers();
+      if (!manager.isEmpty()) {
+        writeCheckpointFile(manager);
+        LOGGER.info("Checkpoints for the current operation were written to {}.", CHECKPOINT_CSV);
+      }
```

Comment:
Does it make sense to add another line in the log with something like the following, so that the user won't have to look up in the manual on how to leverage this checkpoint file?

`LOGGER.info("To resume the current operation, add the following command line flag --dsbulk.log.checkpoint.file={} .", CHECKPOINT_CSV);`

Reply:
Yes, good idea. I also think we should skip this message completely if the operation completed successfully, wdyt? It doesn't make any sense to resume a completed operation.

Reply:
Agreed. If the job didn't get interrupted, it will be confusing to see this message.